cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -121,6 +121,7 @@ func TestAddFlags(t *testing.T) {
 		"--storage-backend=etcd3",
 		"--service-cluster-ip-range=192.168.128.0/17",
 		"--lease-reuse-duration-seconds=100",
+		"--compatibility-version=v1.29",
```

Comment:
do we want to encode the compat version? what happens when this falls out of skew?

Reply:
We can use `version.Get()` and decrement `Minor()` no?

Reply:
`version.Get()` minor is 0, and there is no further decrement from 0. I could not find any method to override that.

Reply:
I think you need to pull git tags to get the real version.

Reply:
Also, yes there isn't a helper method, but I was imagining just wrapping Minor with decrement functionality.